packages/mobile/src/components/details-tile/DetailsProgressInfo.tsx

```diff
@@ -0,0 +1,113 @@
+import type { SearchTrack, Track } from '@audius/common'
+import {
+  formatLineupTileDuration,
+  playbackPositionSelectors
+} from '@audius/common'
+import { View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import IconCheck from 'app/assets/images/iconCheck.svg'
+import Text from 'app/components/text'
+import { makeStyles } from 'app/styles'
+import { useThemeColors } from 'app/utils/theme'
+
+import { ProgressBar } from '../progress-bar'
+
+const { getTrackPosition } = playbackPositionSelectors
+
+const messages = {
+  played: 'Played',
+  unplayed: 'Unplayed',
+  inProgress: 'In Progress',
+  timeLeft: 'left'
+}
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  progressInfoSection: {
+    gap: spacing(2),
+    width: '100%',
```

Comment:
i think all good with width: 100% but wondering why flexGrow or something wouldn't work instead? i think we have seen weird behavior with width: 100% sometimes on mobile, so i personally try to avoid it when i can